CHANGELOG.md

```diff
@@ -250,6 +250,7 @@ This release contains the activation blocks for London across all supported test
 - Change Ethstats's status from experimental feature to stable. [\#2405](https://github.com/hyperledger/besu/pull/2405)
 - Fixed disabling of native libraries for secp256k1 and altBn128. [\#2163](https://github.com/hyperledger/besu/pull/2163)
 - eth_feeHistory API for wallet providers [\#2466](https://github.com/hyperledger/besu/pull/2466)
+- Add option to require replay protection for locally submitted transactions [\#1975](https://github.com/hyperledger/besu/issues/1975)
```

Comment:
we would want this changelog entry in the next future version 22.1.0-RC2 :)

Reply:
thanks @macfarla - sorry I've been so slow to get back to this! :D

Reply:
no worries - this was an easy fix!